docker/Arm64CrossCompile.Dockerfile

```diff
@@ -0,0 +1,79 @@
+FROM ubuntu:22.04 as builder
```

Comment:
suggestion: We could create a new directory to maintain the Dockerfile, for example: `docker/aarch64/Dockerfile`.